### DISCUSSION TITLE
React Swiper ignore 'init: false' parameter

### DISCUSSION THREAD

**Discussion by IAluI** [nolimits4web/swiper#4754] [Q&A]
Hi!

I noticed that starting from version 6.4.9 Swiper stopped taking into account the "init: false" parameter.
 
Is this a bug or a feature?

For example: [https://codesandbox.io/s/swiper-init-bug-lc0p7](https://codesandbox.io/s/swiper-init-bug-lc0p7)

**Comment by nolimits4web** [comment]
Yes, it ignores it because it is required for Swiper React components to work. Any reason to use it and not init actually? Don't want it to be enabled, use `enabled` prop

**Reply by IAluI** [reply]
I am using Puppeteer to generate html. This html is further used for SSR. If I allow Swiper to initialize while the application is being run in Puppeteer, then Swiper modifies the DOM. This is not a desirable effect for me, because it creates additional problems at the stage of hydration.

If it is difficult on your part to maintain the "init" parameter in React Swiper, then problems like mine can be solved with conditional rendering. By creating your own component that creates the same DOM as not initalized Swiper. And output it on the first render.

Do you agree with me that it is impossible to hydrate the html received by an initialized Swiper?)

I would be glad to hear comments regarding my approach. Thanks for the quick response!
